src/hooks/useCampPartygoers.ts

```diff
@@ -1,30 +1,49 @@
-import { useMemo } from "react";
+import { useCallback, useMemo } from "react";
 
 import { User } from "types/User";
 import { WithId } from "utils/id";
 import { partygoersSelector } from "utils/selectors";
 
 import { useConnectPartyGoers } from "./useConnectPartyGoers";
+import { useConnectRelatedVenues } from "./useConnectRelatedVenues";
 import { useSelector } from "./useSelector";
 import { useUserLastSeenThreshold } from "./useUserLastSeenThreshold";
+import { useVenueId } from "./useVenueId";
 
 /**
  * Hook to retrieve a list of partygoers who were in venueName
  * within lastSeenThreshold.
  *
  * @param venueName venueName to filter partygoers by
  */
-export const useCampPartygoers = (venueName: string): WithId<User>[] => {
+export const usePartygoersThreshold = (venueName: string): WithId<User>[] => {
```

Comment:
The filename should be changed to align with what we use for the hook here.

In https://github.com/sparkletown/sparkle/pull/840 I called it `useVenueRecentPartygoers` ([ref](https://github.com/sparkletown/sparkle/pull/840/files?file-filters%5B%5D=.ts&hide-deleted-files=true#diff-ecf886182db52ebfa05dccb6dca7f89aaa06557ab1e5ef68d5fde0d54b31acad)), but `useRecentVenuePartygoers` sounds better.

I don't have a strong preference what name we use, so long as it describes the data/purpose well.

Reply:
Noted.